mvc/src/main/resources/Texts.properties

```diff
@@ -26,4 +26,19 @@ Size.productCreate.name=The product name must be between 3 and 50 characters lon
 NotNull.productCreate.price=The price must not be empty.
 typeMismatch.productCreate.price=The price must be a number.
 Min.productCreate.price=The price must be greater than zero.
-ProductCreateDTOValidator.expensive.black.product=It is too expensive for a black product, should be less than 100.
\ No newline at end of file
+ProductCreateDTOValidator.expensive.black.product=It is too expensive for a black product, should be less than 100.
+
+navigation.driver=Driver
```

Comment:
I don't think we need to use this Texts.properties since we don't want to have multiple language support